airtable/test_airtable_to_mailing_list.py

```diff
@@ -0,0 +1,81 @@
+"""Test airtable_to_mailing_list.py"""
+from airtable_to_mailing_list import *
+import unittest
+
+class TestAirtableToMailinglist(unittest.TestCase):
+    def test_create_chapter_to_mailing_list(self):
+        chapters = [{"id": "one", "fields": {"Mailing List Email": "one@email.com"}},
+                    {"id": "two"}]
+        c_t_ml = create_chapter_to_mailing_list(chapters)
+        self.assertEqual(c_t_ml, {"one": "one@email.com"})
+
+    def test_get_members_to_add(self):
+        members = [
+            {
+                "id": "one",
+                "fields": {
+                    "email": "blah@blah.com",
+                    CHAPTER_ID_COL: ["one_chapter"]
+                }
+            },
+            {
+                "id": "two",
+                "fields": {
+                    "email": "nah@gmail.com",
+                    CHAPTER_ID_COL: ["two_chapter"],
+                    ADDED_TO_MAILING_LIST_COL: ["two_chapter"]
+                }
+            },
+            {
+                "id": "three",
+                "fields": {
+                    "email": "nope@email.com",
+                    CHAPTER_ID_COL: ["three_chapter"],
+                    OPTED_OUT_COL: True
+                }
+            },
+            {
+                "id": "four",
+                "fields": {
+                    CHAPTER_ID_COL: ["four_chapter"]
+                }
+            },
+            {
+                "id": "five",
+                "fields": {
+                    "email": "yup@email.com",
+                    CHAPTER_ID_COL: ["one", "two", "three"],
+                    ADDED_TO_MAILING_LIST_COL: ["one"]
+                }
+            }
+        ]
+        to_add = get_members_to_add(members)
+        # check "one"
```

Comment:
previous convo disappeared from this view because of the rename, but 
> turn both the expected output and actual output into structures that don't have an ordering, so like a set, so then you can test their equality without order mattering